### ISSUE TITLE
Refactor the guide to user registration flow

### ISSUE THREAD

**Issue by sarahmaddox** [kubeflow/website#1614]
The guide to the user registration flow needs some attention. It needs to follow our style guide. 
https://www.kubeflow.org/docs/components/central-dash/registration-flow/

Also consider how best to flow from this guide to the more indepth guide on multi-user isolation mentioned in issue https://github.com/kubeflow/website/issues/1613. Consider whether this page becomes redundant or should be merged into one of the new pages we may create.

**Comment by simmscg** [comment]
I'll take a look at this one.

**Comment by sarahmaddox** [comment]
Thanks @simmscg! Does that mean you're also up to tackling https://github.com/kubeflow/website/issues/1613? I think they may go hand in hand.

**Comment by simmscg** [comment]
Oh gosh, TBA. I'm not sure if #1613 will fit into the sprint time that's left. In saying that, #1614 looks like it needs a lot of extra context, so I think you're right. I'll do a bit more reading.

**Comment by simmscg** [comment]
After talking with Sarah, I'm going to keep these both on unassigned -- there are a few steps that need to happen before they get tackled, and they may not happen before the doc sprint finishes.

**Comment by sarahmaddox** [comment]
I'll take this one on.
